choccake-react/src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Header from './components/Header.js';
+import Footer from './components/Footer.js';
+import Recipe from './components/Recipe.js'; 
+import Image from './components/ImageSection.js';
+import ingredients from './data.js';
+import IngredientsList from './components/IngredientList.js';
+import './App.css';
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <>
+        <Header />
+        <div>
+          <Image />
+          <Recipe />
+        </div>
+        {
+          ingredients.map(ingredient =>
```

Comment:
This map should happen inside the IngredientsList component.

The only prop you should be passing down here is the ingredients, like data = {ingredients}.